scripts/zones/Beadeaux_[S]/mobs/Da_Dha_Hundredmask.lua

```diff
@@ -1,7 +1,7 @@
 -----------------------------------
 -- Area: Beadeaux [S] (92)
 --  NM:  Da'Dha Hundredmask
--- @spawnmob 17154195
+-- @spawnmob 17154095
```

Comment:
ya rest looks alright. If I missed somethin we can fix it later, nothing stands out.